Release/1.0.0

cs-ankit-gupta opened the pull request:
Following changes are made:
1. Removed PB: “SysAdmin User Onboarding Request”, “SysAdmin User Offboarding Request”, “DbAdmin User Onboarding Request” and “DbAdmin User Offboarding Request” and added single playbook “Scenario - Employee On/Off Boarding” for Simulation
2. Removed global variable “EOF_SYS_User” and “EOF_SYS_User”. In place of this added two variable in configuration step of PB “Scenario - Employee On/Off Boarding”.
3. Added docs in latest format.
4. Added Configuration Step in each PB.

cs-ankit-gupta committed:
Made changes

cs-ankit-gupta committed:
Docs

cs-ankit-gupta committed:
Delete .DS_Store

cs-ankit-gupta committed:
Changes as mentioned below

Following changes are made:
	1. Removed PB: “SysAdmin User Onboarding Request”, “SysAdmin User Offboarding Request”, “DbAdmin User Onboarding Request” and “DbAdmin User Offboarding Request” and added single playbook “Scenario - Employee On/Off Boarding” for Simulation
	2. Removed global variable “EOF_SYS_User” and “EOF_SYS_User”. In place of this added two variable in configuration step of PB “Scenario - Employee On/Off Boarding” where user can add name as per there requirement for Simulation.
	3. Added docs in latest format.
	4. Added Configuration Step in each PB.

cs-ankit-gupta committed:
Merge branch 'release/1.0.0' of https://github.com/cs-ankit-gupta/solution-pack-employee-onboarding into release/1.0.0

cs-ankit-gupta committed:
Added Tag and Category

Commit:
release/1.0.0

1. Added Authored By in Readme.md
2. Added publisher as Fortinet SE
3. Made some content correction

cs-ankit-gupta committed:
Delete .DS_Store

cs-ankit-gupta committed:
Update .gitignore

cs-abhishek-shukla committed:
1.0.0 Changes

cs-ankit-gupta committed:
Merge pull request #1 from cs-ankit-gupta/1.0.0-doc-changes

1.0.0 Changes

cs-suyog-jinturkar commented on docs/contents.md:
```diff
@@ -0,0 +1,46 @@
+
+| Name                       | Description                                                                                                                                                                                      |
+|:---------------------------|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Excahnge                   | The Exchange connector provides a robust, platform-independent, and simple interface for communicating with Microsoft Exchange 2007-2016 Server or Office 365 using Exchange Web Services (EWS). |
```
spelling mistake

cs-ankit-gupta committed:
Update contents.md

cs-ankit-gupta committed:
Update info.json

cs-suyog-jinturkar reviewed (commented):
added review comments

Commit:
Coorection

Corrected SOAR Framework version to 1.1.0 and changed Simulate Scenario Documentation link to develop

cs-suyog-jinturkar reviewed (approved):
Assuming @sujit has reviewed the contents of PBs, modules, etc and Ankit has unit tested this throughly. Approving this PR.
